endpoints/api/namespacequota.py

```diff
@@ -31,12 +30,18 @@
 logger = logging.getLogger(__name__)
 
 
-def quota_view(quota):
+def quota_view(quota, default_config=False):
     quota_limits = list(model.namespacequota.get_namespace_quota_limit_list(quota))
 
+    # If no quota is defined for the org, return systems default quota if set
+    if not quota and config.app_config.get("DEFAULT_SYSTEM_REJECT_QUOTA_BYTES") != 0:
```

Comment:
if quota could possibly be non you need to check earlier before you pass it to namespace_quota_limit_list as that would now be a unnecessary operation that can be avoided. Otherwise this check is not necessary

Reply:
Yup, makes sense. I'll check for this before the method `get_namespace_quota_limit_list` is called.

Reply:
the none check is fine just move it up to be the first thing quota is used for.

Reply:
Done